subprojects/core/src/main/java/org/gradle/internal/fingerprint/impl/ClasspathFingerprintingStrategy.java

```diff
@@ -77,6 +83,14 @@ public ClasspathFingerprintingStrategy(NonJarFingerprintingStrategy nonJarFinger
         this.jarHasherConfigurationHash = hasher.hash();
     }
 
+    static ClasspathFingerprintingStrategy runtimeClasspath(ResourceFilter classpathResourceFilter, RuntimeClasspathResourceHasher runtimeClasspathResourceHasher, ResourceSnapshotterCacheService cacheService, StringInterner stringInterner) {
```

Comment:
Can we make the two factory methods public? We don't seem to use package private visibility in the rest of the code base.